### ISSUE TITLE
How to control pause/resume trough options/command line?

### ISSUE THREAD

**Issue by matteoraggi** [popcornmix/omxplayer#534]
Is it possible to pause and resume omxplayer trough command line? I can't find it in the command line options... 
Thanks for the patience.

**Comment by pcwalden** [comment]
'p' pauses and resumes

**Comment by matteoraggi** [comment]
P is pausing and resuming if I press manually on the keyboard the P button, but trough command line I can't press P, because I can't use the same terminal session while playing, am I right?

**Comment by kennyyy24** [comment]
You can use dbuscontrol.sh script to do this:
$ ./dbuscontrol.sh pause 
should do what you want. If you are after a bit more control (pause only or play only for instance) you can tweek the dbuscontrol.sh with other available dbus controls.

**Comment by bholmes451** [comment]
I have added a command line parameter in my version which allows you to start the video paused and then resume via keyboard command or dbuscontrol.  I'm currently attempting to get a pull request figured out to merge it into the main code. Is that what you wanted to do?

**Comment by matteoraggi** [comment]
Yes, it is a great idea! is it possible also to configure for how much time the video would remain in pause directly?

**Comment by bholmes451** [comment]
I hadn't considered that as my use case is specifically for commanded resume but I can look into it.

**Comment by matteoraggi** [comment]
Very good, thanks! I am very courious about it, I would be happy to be your alfa and beta tester :-)

**Comment by jordiblanchcarles** [comment]
Hello, does somebody know if the command line parameter to start omxplayer in pause has already been included in omxplayer stable version, or there's any chance it will? I'm very interested in this parameter because I'd like to start an omxplayer instance to play a video and meanwhile start another one in background that should load the video and keep in pause until the first one quits. I've tried to do this using named pipes but sometimes it is not working properly and I get the 2 videos playing at the same time.

Thank you in advance!

**Comment by bholmes451** [comment]
I have just figured out what was going on with my fork and submitted a pull request for this enhancement. If you need this immediately you can compile from my fork (which has been updated to match the most recent popcornmix version: https://github.com/bholmes451/omxplayer/

**Comment by matteoraggi** [comment]
veyr nice! I found the command "--start-paused          Immediately pause the video after loading, will wait for dbus or key command to play" so after starting pauses, which is the command to play the video then?

**Comment by jordiblanchcarles** [comment]
Perfect! Thank you! I'll try your fork. Thank you very much!

**Comment by bholmes451** [comment]
This is the only change between the current popcornmix version and my fork. I did a pull this morning from the popcornmix master branch. You can see the changes here: https://github.com/popcornmix/omxplayer/pull/538

Basically I changed the code from automatically starting playback after the buffer was filled to pausing when the flag is sent.  You can use the p key or the space key.  The dbus command I have used is `./dbuscontrol.sh play` or `./dbuscontrol.sh pause`
